static/src/stylesheets/module/football/_matches.scss

```diff
@@ -141,9 +141,25 @@
     cursor: pointer;
 }
 
+$show-more-icon-size: $base-size * 3;
+$show-more-icon-spacing: $base-size + 1;
+
 .football-matches__show-more {
     margin-top: $gs-baseline/2;
     outline: none;
+    position: relative;
+    padding-left: $show-more-icon-size + $show-more-icon-spacing * 2;
+
+    .i, svg {
+        width: $show-more-icon-size;
+        height: $show-more-icon-size;
+        vertical-align: middle;
+        margin: auto;
+        position: absolute;
+        top: 0;
+        bottom: 0;
+        left: $show-more-icon-spacing;
+    }
```

Comment:
I am not sure those changes are related

Reply:
With javascript disabled, which is how integration tests run, the football `show more` button is being hidden, even though it shouldn't be.
With former version of Selenium (brought within Play 2.5), clicking an hidden element during tests was not a problem however new version of Selenium prevents any interactions with an element that is not visible.
This fix ensures:
- the button is visible
- the integration tests checking the behaviour of this button works

There is a cleaner way to fix this issue but it involves touching all `show more` button throughout the website. I will implement it in a subsequent PR